index.js

```diff
@@ -157,9 +157,15 @@ Client.prototype.validateXmlrpcVersion = function( callback ) {
 					"XML-RPC extensions for Gilded WordPress are not installed." ) );
 			}
 			if ( !error.code ) {
-				return callback( new Error( "Unknown error. " +
+				var msg = "Unknown error. " +
```

Comment:
*We* don't know what the error is, but if we're providing the original error object, then there's enough context that we should no longer include this sentence.